PICMI_Python/particles.py

```diff
@@ -39,11 +39,11 @@ def __init__(self, particle_type=None, name=None, charge_state=None, charge=None
 
         self.handle_init(kw)
 
-    def activate_ionization(self, model, target_species):
+    def activate_field_ionization(self, model, product_species):
```

Comment:
There should be a target and a product (or emitted) species. Presumably, the self is the incident species.

Though, I think more thought is needed on how to deal with interactions. I don't think it should be part of a species since it involves multiple species.

Reply:
I agree that the interface for atomic/particle interactions requires more thought. 
For now, in the current PR, I will keep it as is.